### DISCUSSION TITLE
How to enable New UI / Ambient mode

### DISCUSSION THREAD

**Discussion by TROLEX69** [PoomSmart/YTABConfig#63] [Q&A]
So when ever I install the app for the first time and sign in, it loads the new YouTube UI. After closing the app from the background and starting it again it loads back to the UI and loses the ambient mode also. How can I re enable the new UI? Any help would be appreciated. 

Old UI I’m having right now vs New UI I’m trying to get:
https://images.app.goo.gl/gJo3UYkEbNAQQ4dK8

**Comment by arichornlover** [comment]
Hmm it’s actually easy, I’m pretty much a master at this sort of thing and can easily get you the new ui.
Enable the following flags below
`enableCinematicContainer`
`enableCinematicContainerOnClient`
`enableCinematicContainerOnTablet`
`iosCinematicContainerClientImprovement`
`mainAppCoreClientEnableClientCinematicTablets`
`mainAppCoreClientEnableClientCinematicPlaylists`
`uiSystemsClientGlobalConfigEnableRoundedDialogForNative`
`uiSystemsClientGlobalConfigEnableRoundedThumbnailsForNative`
`uiSystemsClientGlobalConfigEnableRoundedTimestampForNative`
`uiSystemsClientGlobalConfigEnableModernButtonsForNative`
`modernizeElementsBgColor`
`uiSystemsClientGlobalConfigModernizeNativeBgColor`

These are the flags. Hope this helps! I’m sorry that PoomSmart didn’t help you but glad I stepped in!

**Reply by TROLEX69** [reply]
Thank you for the help, although after setting these flags I got the ambient mode option, but unfortunately I didn’t get the new UI nor did the ambient mode actually work. 
Screenshots while the video running:
![9C0BE5AB-62BB-4EC7-83A2-2EE371B56A74](https://user-images.githubusercontent.com/64669383/221397963-6f500d8a-8ac1-4d8e-9a73-8b16c889cb19.png)

Screenshot for the menu option:
![9C34FB46-8015-4B9E-BA82-6E528EB04060](https://user-images.githubusercontent.com/64669383/221398020-b5af89fd-1b50-414d-a4b1-657bc4da7eff.png)

Screenshot for modified flags:
![1BEDAD5F-38DA-4390-98F4-3995FFD75B7D](https://user-images.githubusercontent.com/64669383/221398042-6e779058-5594-436b-8701-a51958a8d86b.png)

**Reply by arichornlover** [reply]
That is a known problem. YouTube has the new ui for the video player client side and there is no flag to toggle the new video player layout. 
![897EE77E-710A-4251-9FEA-AD1056526047](https://user-images.githubusercontent.com/78001398/221398291-a3221d64-a415-4a2f-a42a-f5c79518708f.jpeg)
If it still doesn’t work after having this layout since it’s important for it then that pretty much means YouTube most likely broke the whole Ambient Mode feature on mobile.

**Comment by PoomSmart** [comment]
> I’m sorry that PoomSmart didn’t help you

My responsibility for YTABConfig is to add a feature or fix a bug inside the tweak itself. The exploration of A/B flags is up to users, as intended by the nature of this tweak.

Here you are again, arichorn, being too verbose, making assumptions and crossing the line multiple times.

This is a final warning. You will be banned from all of my YouTube projects next time. Consider this my last bit of sympathy for even caring about what an internet guy would do.
